cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -109,6 +110,9 @@ public TestCommand(Path projectPath, PrintStream outStream, PrintStream errStrea
     @CommandLine.Option(names = "--jacoco-xml", description = "enable Jacoco XML generation")
```

Comment:
Are we still supporting the `--jacoco-xml` flag or are we planning on removing it later? Would this be covered by `--coverage-format`?

Reply:
We will be supporting it to avoid the stdLib builds from going through a breaking change. Will be removing it in the next iteration. Created https://github.com/ballerina-platform/ballerina-lang/issues/30622 to track this.